maintainers/maintainer-list.nix

```diff
@@ -3951,6 +3951,12 @@
     githubId = 2502736;
     name = "James Hillyerd";
   };
+  jiehong = {
+    email = "nixos@majiehong.com";
+    github = "Jiehong";
+    githubId = 42 ?;
```

Comment:
How do you get that value?

Reply:
You can retrieve it via API from https://api.github.com/users/Jiehong (as mentioned at the top of the file).

Reply:
@pbogdan : thanks!